src/measures.tex

```diff
@@ -233,32 +230,69 @@ \subsection{Spaces of Measures}
 \begin{proof}
   \uses{theta-y}
   \leanok
-  Pick $x\in \R$, and consider the power series
+  Given $n\in\N$ and $x\in\R$, set
+	\[
+		Y_\xi(x, n)=
+		\begin{cases}
+			x&\text{ if } n = 0 \\
+			Y_\xi(x,n-1) - \left\lfloor \frac{Y_\xi(x,n-1)}{\xi^{n-1}} \right\rfloor \xi^{n-1} & \text{ if } n \ge 1
+		\end{cases}
+	\]
+where $\lfloor \ast \rfloor$ denotes the floor of $\ast$. Consider the power series
   \[
-    F(T)=\sum_{n\geq 0}\left\lfloor \frac{Y_\xi(x,n)}{\xi^n}\right\rfloor T^n\in\Z[\![T]\!]
+    F(T)=\sum_{n\geq 0}\left\lfloor \frac{Y_\xi(x,n)}{\xi^n}\right\rfloor T^n\in\Z[\![T]\!].
   \]
-  where $\{Y_\xi(x,n)\}_{n}$ is the sequence defined in Definition~\ref{theta-y}. The fact that $0<\xi<1$ ensures that the series converges on the open unit disk, and it can be proven that $F(\xi)=x$.
+	The fact that $0<\xi<1$ ensures that the series converges on the open unit disk, so it actually belongs to $\Z(\!(T)\!)_{r'}$ and it can be proven that $F(\xi)=x$.
 %  By Proposition~\ref{theta.has_sum_x}, the series converges on the open unit disks,
 %  so it belongs to $\Z((T))_r$ and it specializes to $x$ when evaluated at $T=\xi$.
 \end{proof}
+The next lemma allows to compare the seemingly decorrelated topologies on the spaces of \emph{real} and of \emph{Laurent} measures:
+\begin{lemma}
+	\label{theta.continuous}
+	\uses{theta}
+	\uses{laurent-measures-fin}
+	\uses{real-measures}
+	\lean{ses.continuous_θ_c}
+	\leanok
+%Let $0<p'<1$ be a real number, let $S$ be a finite set, and let $r'$ denote $\left(\frac{1}{2}\right)^{p'}$. 
+Fix a finite set $S$, real numbers $0<\xi,p'<1$ and set $r'=(\xi)^{p'}$. The map $\vartheta_\xi$ of Definition~\ref{theta} is continuous when endowing $\Lm_{r'}(S)$ with the topology defined in Definition~\ref{laurent-measures-fin} and $\Rm(S)$ with the topology defined in Definition~\ref{real-measures}
+%\[
+%\theta_{S}\colon \Lm_{r'}(S)\longrightarrow \Rm(S)
+%\]
+%induced by evaluating $T$ at $1/2$ is continuous
+\end{lemma}
+\begin{proof}
+  \leanok
+  Omitted for now, but done in Lean.
+\end{proof}
 
-Given any \emph{finite} set $S$, the above map $\vartheta_{\xi}$ defines a map $\vartheta_{\xi,S}\colon \Lm_{r'}(S)\longrightarrow \Rm(S)$ by extending $\vartheta_{\xi}$ componentwise.
-For the purpose of this project, the special case where $\xi=1/2$ is enough, and we write $\theta_S$ to denote $\vartheta_{\xi,S}$. 
+Given any finite set $S$, the map $\vartheta_{\xi}$ defines a map $\vartheta_{\xi,S}\colon \Lm_{r'}(S)\longrightarrow \Rm(S)$ by extending $\vartheta_{\xi}$ componentwise. For the purpose of this project, the special case where $\xi=1/2$ is enough, and we write $\theta_S$ from now on to denote $\vartheta_{\xi,S}$. 
 
-\begin{lemma}
-  \label{theta-continuous}
+%By taking projective limits, we obtain a map $\vartheta_{S}$ for every profinite $S$.
+
+\begin{theorem}
+  \label{theta.exact_finite}
```

Comment:
Idem. Maybe also elsewhere?

Reply:
I have checked all of them in the measures.tex file.